src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -623,6 +623,36 @@ private function step_in_body() {
 		$op_sigil = $this->is_tag_closer() ? '-' : '+';
 		$op       = "{$op_sigil}{$tag_name}";
 
+		if ( null === $tag_name && '#text' === $this->get_token_type() ) {
+			/*
+			 * This rule is necessary even without supporting text nodes in the
+			 * HTML Processor because the parser has to move past text nodes, and
+			 * there could be breadcrumb implications when the text triggers the
+			 * active format reconstruction.
+			 *
+			 */
+			$this->reconstruct_active_formatting_elements();
+
+			$current_token = $this->bookmarks[ $this->state->current_token->bookmark_name ];
+			if ( 1 === $current_token->length && "\x00" === $this->html[ $current_token->start ] ) {
+				// Parse error: ignore the token.
+				return $this->step();
+			}
+
+			/*
+			 * Whitespace-only text does not affect the frameset-ok flag.
+			 * It is probably inter-element whitespace, but it may also
+			 * contain character references which decode only to whitespace.
+			 */
+			$text = $this->get_modifiable_text();
+			if ( strlen( $text ) !== strspn( "\t\n\f\r", $text ) ) {
```

Comment:
A character token that is one of U+0009 CHARACTER TABULATION, U+000A LINE FEED (LF), U+000C FORM FEED (FF), U+000D CARRIAGE RETURN (CR), or U+0020 SPACE

It looks like this is missing " ":

```suggestion
			if ( strlen( $text ) !== strspn( "\t\n\f\r ", $text ) ) {
```

Reply:
thanks. I thought I had that in there when I wrote it

Reply:
fixed in 5b6e461bc544dbb4a8ad64f35e09631c7aed855a
we both missed that the order of arguments was backwards, though since this only affects the `frameset-ok` flag we wouldn't have seen a bug _yet_